s3plugin/s3plugin.go

```diff
@@ -99,6 +100,20 @@ func readAndValidatePluginConfig(configFile string) (*PluginConfig, error) {
 	return config, nil
 }
 
+func SetLoggerVerbosity(loggerVerbosity string) {
+	gplog.SetLogFileVerbosity(gplog.LOGINFO)
```

Comment:
shouldn't we add here (for default case)
```
gplog.SetVerbosity(gplog.LOGINFO)
```
?

Reply:
No, it is already [initialized with the info level](https://github.com/arenadata/gp-common-go-libs/blob/3ce86b17fa4fb3732d7a987ef6bb2e33ce281ca0/gplog/gplog.go#L139).